bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueSensorHandler.java

```diff
@@ -225,6 +261,21 @@ public void onSensorStateChanged(@Nullable HueBridge bridge, FullSensor sensor)
                 // do nothing
             }
         }
+        
+        Object status = sensor.getState().get(STATE_STATUS);
+        if (status != null) {
+            try { DecimalType value = new DecimalType(String.valueOf(status));
+            updateState(STATE_STATUS, value);} catch (DateTimeParseException e) {
```

Comment:
Why do you expect `DateTimeParseException` when paring a `DecimalType`? I would guess that `NumberFormatException` should be catched here.

Reply:
modified in new commit